libs/blocks/accordion/accordion.js

```diff
@@ -12,59 +13,98 @@ function setSEO(questions) {
   document.head.append(script);
 }
 
-function toggleMedia(con, trig, status) {
-  if (status === 'open') {
-    trig.setAttribute('hidden', '');
-    trig.setAttribute('aria-expanded', 'false');
-    con.setAttribute('hidden', '');
-  } else {
-    trig.setAttribute('aria-expanded', 'true');
-    trig.removeAttribute('hidden');
-    con.removeAttribute('hidden');
-  }
+/* c8 ignore next 8 */
+function playVideo(video) {
+  if (!video) return;
+  if (video.getAttribute('autoplay') === null) return;
+  const playBtn = video.nextElementSibling;
+  const isPlaying = playBtn.getAttribute('aria-pressed') === 'true';
+  if (isPlaying || video.readyState === 0) return;
+  playBtn.click();
 }
 
-function displayMedia(displayArea, el, dd, i, expanded) {
-  const id = el.getAttribute('aria-controls').split('-')[1];
-  [...mediaCollection[id]].forEach(
-    (mediaCollectionItem, idx, total) => {
-      mediaCollectionItem.classList.remove('expanded');
+/* c8 ignore next 11 */
+function pauseVideo(video) {
+  if (!video) return;
+  if (video.getAttribute('controls') !== null) {
+    video.pause();
+    return;
+  }
+  const pauseBtn = video.nextElementSibling;
+  const isPlaying = pauseBtn?.getAttribute('aria-pressed') === 'true';
+  if (!isPlaying || video.readyState === 0) return;
+  pauseBtn.click();
+}
 
-      total.forEach((element, index) => {
-        const trigger = document.querySelector(`#accordion-${id}-trigger-${index + 1}`);
-        const content = document.querySelector(`#accordion-${id}-content-${index + 1}`);
-        toggleMedia(content, trigger, 'open');
-      });
-      toggleMedia(dd, el);
-      displayArea.childNodes[i - 1].classList.add('expanded');
-
-      if (expanded) {
-        toggleMedia(dd, el, 'open');
-        displayArea.childNodes[i - 1]?.classList.remove('expanded');
-      }
-    },
-  );
+function openPanel(el, dd) {
```

Comment:
I see that the `dd` param name was used in the code that was already here, but it might be good to use a more human readable name.